PlayCover/Model/PlayApp.swift

```diff
@@ -112,6 +116,7 @@ class PlayApp: BaseApp, ObservableObject {
 
     func clearAllCache() {
         Uninstaller.clearExternalCache(info.bundleIdentifier)
+        Uninstaller.clearExternalCache(info.bundleName)
```

Comment:
This line does not need to be here. The name of the folder of all the caches are always the bundle identifier.